engine/test/.transform.js

```diff
@@ -0,0 +1,28 @@
+const Babel = require('@babel/core')
```

Comment:
One for interests sake; What is the purpose of these files, what do they do for us and why?

Reply:
I was having some trouble un-hoisting node_modules/@xgovformbuilder/lab-babel, so I've got the transform scripts in here. I'll refactor into model or a new shared /lib and remove @xgovformbuilder/lab-babel